src/nav2_localization/amcl/CMakeLists.txt

```diff
@@ -20,18 +20,14 @@ find_package(sensor_msgs REQUIRED)
 find_package(std_srvs REQUIRED)
 find_package(tf2_ros REQUIRED)
 find_package(tf2 REQUIRED)
-#find_package(util REQUIRED)  //TODO (mhpanah) remove ../util/include
+find_package(nav2_util REQUIRED)
 
 include_directories(
   include 
-  ../util/include
+ # ../util/include
```

Comment:
Why not remove this line instead of comment it out?

Reply:
Yes.  This should have been deleted.

Reply:
Weird, when I go in to edit mode the line of code isn't there. Do you see it in your source files?